src/account_sentinel.rs

```diff
@@ -66,6 +66,72 @@ impl<Request, Name, Claim> AccountSentinel<Request, Name, Claim>
         let mut claims = map.iter().map(|(_, ref claim)| claim.clone())
                             .collect::<Vec<_>>();
         claims.sort();
-        Some(claims[claims.len() / 2].clone())
+        Some(claims[(claims.len() - 1) / 2].clone())
```

Comment:
No, this is on purpose:

sequence | previous calculation | new calculation
--------------|----------------------------|--------------------
[0]             | 0                              | 0
[0,1]          | 1                              | 0
[0,1,2]       | 1                              | 1
[0,1,2,3]    | 2                              | 1
[0,1,2,3,4] | 2                              | 2